src/pages/ValidatorOnboarding/ContinueButton.tsx

```diff
@@ -144,13 +129,15 @@ const ContinueButton = () => {
       <XStack
         style={{
           width: '100%',
-          justifyContent: isActivationValScreen ? 'space-between' : 'end',
+          justifyContent:
+            isActivationValScreen || true ? 'space-between' : 'end',
```

Comment:
Doesn’t this always return true? Then what is the point of the ternary operator

Reply:
I was testing and forgot about that